plugins/inputs/ping/ping.go

```diff
@@ -262,7 +277,7 @@ func (p *Ping) pingToURLNative(destination string, acc telegraf.Accumulator) {
 
 	sort.Sort(durationSlice(stats.Rtts))
 	for _, perc := range p.Percentiles {
-		var value = percentile(durationSlice(stats.Rtts), perc)
+		var value = percentile(stats.Rtts, perc)
```

Comment:
I am correct, that you dropped the durationSlice since it happens two lines up?

Reply:
I dropped `durationSlice` because it was redundant type conversion. Unfortunately it is not possible to remove it from earlier occurence (compilation error).